### PR TITLE
Add support for -package-name

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

**Comment by keith** [comment]
i don't think there's a need to gate this on swift version since if you actually use this on a target, and then use the package ACL, that would fail anywayas

File: swift/internal/attrs.bzl [hunk 1]
```diff
@@ -129,6 +129,12 @@ The name of the Swift module being built.
 If left unspecified, the module name will be computed based on the target's
 build label, by stripping the leading `//` and replacing `/`, `:`, and other
 non-identifier characters with underscores.
+""",
+            ),
+            "package_name": attr.string(
+                doc = """\
+The semantic package of the Swift target being built. Targets with the same
```
Comment (brentleyjones) [inline swift/internal/attrs.bzl:136]: We should mention that it required Swift 5.9+

**Review by brentleyjones — APPROVED** [review]

**Comment by keith** [comment]
sg

**Comment by keith** [comment]
FWIW  this does work now, we can still wait until a bit closer to merge tho

**Comment by brentleyjones** [comment]
Is this in an Xcode beta? If so, I think we can merge now.

**Comment by keith** [comment]
yea it is, sg

**Pull request merged**